tests/51-live-user_notification.c

```diff
@@ -0,0 +1,88 @@
+#include <sys/types.h>
```

Comment:
nitpick - do you want to add your copyright info at the top?

Reply:
Good point, fixed.